server/app/filters/DisableCachingFilter.java

```diff
@@ -22,11 +34,28 @@ public EssentialAction apply(EssentialAction next) {
         request ->
             next.apply(request)
                 .map(
-                    result ->
-                        result
-                            .withHeader("Cache-Control", "no-cache, must-revalidate")
-                            .withHeader("Pragma", "no-cache")
-                            .withHeader("Expires", "0"),
+                    result -> {
+                      Optional<String> pathPrefix = Iterables.getFirst(PATH_SPLITTER.splitToList(request.uri()));
+                      Integer status = result.status();
+
+                      if (environment.isDev()) {
+                        // Must revalidate status asset caches in dev mode
+                        // pathPrefix = Optional.empty();
```

Comment:
question (blocking): commented out?

Reply:
This is out of date - it was disabled for testing